owncloud/ios-app
# [feature/iOS-15-SDK] Build with iOS 15 SDK

## Description
App Store requires the latest iOS 15 SDK for new uploads. 
After upgrading to iOS 15 there were some UI bugs:
- Share Extension: Toolbar color were opaque 
- App: Tabbar, Toolbar color were not applied 

## Related Issue
#1066 

## Motivation and Context
Support the latest iOS SDK (App Store requires iOS 15 SDK now)

## How Has This Been Tested?
- Build with Xcode 13
- Check the mentioned fixes from the description section
- perform tests with all color themes (most fixes were in _Contrast_ theme)
- Re-Check iOS version < iOS 15 too with the same checks above

## Screenshots (if appropriate):

## Types of changes


## Checklist:
